python/sglang/srt/layers/fused_moe/patch.py

```diff
@@ -98,7 +98,9 @@ def fused_moe_forward_native(
     renormalize: bool,
     topk_group: Optional[int] = None,
     num_expert_group: Optional[int] = None,
+    custom_routing_function: Optional[Callable] = None,
 ) -> torch.Tensor:
+    assert custom_routing_function is None
```

Comment:
why add this?

Reply:
the new vllm version added this argument